docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: "3.9"
+services:
+  db:
+    image: postgres
```

Comment:
does it make sense to specify the version that we use here to be the same that we use in production? just in case something changes between session, in production we are using `10.17`